Toolset/palettes/start center/revStartCenterBehavior.livecodescript

```diff
@@ -0,0 +1,610 @@
+   if revIDEGetPreference("cStartPageShow") is true then
+      set the iconPathPreset of widget "startupCheckbox" to "lc-checkbox-checked"
+   else
+      set the iconPathPreset of widget "startupCheckbox" to "lc-checkbox-unchecked"
+   end if
+   
+   ## First run
+   if revIDEGetPreference("StartCenterClassic") is true then
+      ## Not first run
+      hide group "firstRun"
+      show group"topPanel"
+      --show group "tiles"
+      set the top of group "recentFiles" to 0
+   else
+      ## First run
+      hide group "banner"
+      hide group "topPanel"
+      --hide group "tiles"
+      
+      set the top of group "recentFiles" to 0
+      
+      set the layer of group "firstRun" to top
+      show group "firstRun"
+   end if
+end openStack
+
+on ideNewStack
+   startCenterClose
+end ideNewStack
+
+on ideOpenStack pTarget
+   if the short name of the owner of pTarget is not the short name of me then
+      startCenterClose
+   end if
+end ideOpenStack
+
+command startCenterClose
+   close me
+end startCenterClose
+
+command logoInitialise
+   ## Set the logo to the correct logo for the version
+   set the iconPathPreset of widget "logo" of me to "lc-logo"
+   set the iconColor of widget "logo" of me to uiColor("logo")
+end logoInitialise
+
+command socialMediaInitialise   
+   ## Set the images
+   repeat for each item tMedia in kAllSocialMedia
+      hide widget tMedia of group "socialMedia"
+      set the iconPathPreset of widget tMedia of group "socialMedia" to ("lc-" & tMedia)
+      set the iconColor of widget tMedia of group "socialMedia" to uiColor(tMedia)
+      set the fillrule of widget tMedia of group "socialMedia" to "even odd"
+      set the blendLevel of widget tMedia of group "socialMedia" to 50
+   end repeat
+end socialMediaInitialise
+
+command socialMediaSet
+   local tCenterLoc, tMediaCount, tPadding, tWidth, tTotalWidth, tLeft
+   
+   ## Show the chosen social media
+   put item 1 of the loc of group "socialMedia" into tCenterLoc
+   put the number of items in kSocialMedia into tMediaCount
+   put 20 into tPadding
+   put 20 into tWidth
+   
+   put (tWidth * tMediaCount) + (tMediaCount - 1) * tPadding into tTotalWidth
+   put tCenterLoc - round(tTotalWidth / 2) into tLeft
+   
+   repeat with x = 1 to tMediaCount
+      show widget (item x of kSocialMedia)
+      set the left of widget (item x of kSocialMedia) to tLeft
+      add tWidth + tPadding to tLeft
+   end repeat
+end socialMediaSet
+
+command socialMediaHilite pName, pValue
+   if pValue is true then
+      set the blendLevel of widget pName of group "socialMedia" to 0
+   else
+      set the blendLevel of widget pName of group "socialMedia" to 50
+   end if
+end socialMediaHilite
+
+command socialMediaUniliteAll
+   repeat with x = 1 to the number of widgets of group "socialMedia"
+       set the blendLevel of widget x of group "socialMedia" to 50
+   end repeat
+end socialMediaUniliteAll
```

Comment:
@runrevelanor Should this handler be called `socialMediaUnhiliteAll` with an "h"?